artist/src/main/java/com/ataulm/artcollector/artist/ui/ArtistActivity.kt

```diff
@@ -33,8 +33,9 @@ class ArtistActivity : AppCompatActivity() {
         recyclerView.adapter = adapter
         recyclerView.layoutManager = GridLayoutManager(this@ArtistActivity, 2)
 
-        viewModel.gallery.observe(this, DataObserver<Gallery> { gallery ->
-            adapter.submitList(gallery)
+        viewModel.artistGallery.observe(this, DataObserver<ArtistGallery> { artistGallery ->
```

Comment:
instead of observing the `gallery` directly, we're now waiting for `artistGallery`, a composite of artist and gallery